src/main/java/duke/Ui.java

```diff
@@ -0,0 +1,34 @@
+package duke;
+
```

Comment:
I think that you can apply some ENUM instances in the Ui class.
```suggestion
public static final String LINE = "------------------------------\n"
```